addon/validators/presence.js

```diff
@@ -1,13 +1,14 @@
 import buildMessage from 'ember-changeset-validations/utils/validation-errors';
 import evValidatePresence from 'ember-validators/presence';
+import { get } from '@ember/get';
```

Comment:
Is the path there `@ember/object`?

Reply:
Of course! Not sure what was going on there :sweat_smile:. I changed it to the correct import path.